### ISSUE TITLE
About training details on train+val split

### ISSUE THREAD

**Issue by woxihuanjiangguo** [HuangJunJie2017/BEVDet#303]
Hello author, 

Are there any modifications on the training process when you train for a high-resolution leaderboard challenge model on train+val split, for example, learning rate, data augmentations or some hyperparameters? 

Thanks!
